compiler/src/iree/compiler/Dialect/VM/Target/Bytecode/BytecodeModuleTarget.cpp

```diff
@@ -533,21 +553,33 @@ static LogicalResult buildFlatBufferModule(
   }
 
   iree_vm_DebugDatabaseDef_ref_t debugDatabaseRef = 0;
-  if (!targetOptions.stripSourceMap) {
+  if (!bytecodeOptions.stripSourceMap) {
     debugDatabaseRef = debugDatabase.build(fbb);
   }
 
   auto moduleNameRef = fbb.createString(
       moduleOp.getSymName().empty() ? "module" : moduleOp.getSymName());
 
+  auto requiredFeatures = findRequiredFeatures(moduleOp);
```

Comment:
good call - today it can be - in the future I'll want to allow functions to require more than the module does but this code isn't doing that so it's just waste!